source/query/atlas-search.txt

```diff
@@ -25,32 +40,27 @@ Steps
    :style: normal
 
    .. step:: Find the index you want to use in your query
-      
-      a. From the :guilabel:`Indexes` tab, click :guilabel:`Search Indexes`.
-      #. Hover over the index you want to use in your query.
 
-   .. step:: Run the query
+      From the :guilabel:`Indexes` tab, click :guilabel:`Search Indexes`. 
+      Hover over the search index you want to use in your query. 
       
-      a. Click the :guilabel:`Aggregate` button.
-
-         |compass| switches to the :guilabel:`Aggregations` tab where you 
-         can run your query. 
-        
-      #. Construct your query.
+   .. step:: Open the aggregation pipeline builder
+      
+      Click the :guilabel:`Aggregate` button that appears when you hover over 
+      your search index.
          
-         By default, |compass| selects the :pipeline:`$search` stage for
-         your query. You can select the  :pipeline:`$searchMeta` stage
-         to use in your query. If you are using a ``vectorSearch`` type
-         index in your query, you can only use the
-         :pipeline:`$vectorSearch` stage in your query.  
+   .. step:: Construct your query
+      
+      If you have an Atlas Search index, add the ``$search`` or 
+      ``$searchMeta`` stage to your aggregation pipeline. If you have an 
+      Atlas Vector Search index, add the ``$vectorSearch`` stage.
```

Comment:
We actually do automatically create the $search or $vectorSearch stage for users if they click the "Aggregate" button from the search indexes list in Compass. Are we modifying this language to make it more compatible with DE? Want to make sure I understand why this change is happening

Reply:
Ah this might've stemmed from some testing I was doing on `cloud-dev` where it didn't automatically add the stage. Is this going to be the case where on DE, they have to add the search stage(s) themselves or will it have the same behavior as Compass?

Reply:
EDIT: Given that this search index management won't be available in DE, it may make more sense to not migrate this page into the DE docs and instead link users to the [Atlas Search documentation](https://www.mongodb.com/docs/atlas/atlas-search/tutorial/run-query/) itself.

Reply:
Thanks Ali!